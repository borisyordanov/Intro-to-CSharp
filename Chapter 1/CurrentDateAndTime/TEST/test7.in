2000
1
2
14
3
5
