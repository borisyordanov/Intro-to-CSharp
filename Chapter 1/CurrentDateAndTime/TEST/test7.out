1/2/2000 2:03:05 PM
